api/api_sources/sources/database/database-schema/schema-sqls/SurveyTypeCodeSchema.sql

```diff
@@ -0,0 +1,39 @@
+-- ### Creating Table: survey_type_code ### --
+
+        
+CREATE TABLE survey_type_code ();
+ALTER TABLE survey_type_code ADD COLUMN survey_type_code_id SERIAL PRIMARY KEY;
+ALTER TABLE survey_type_code ADD COLUMN survey_type_code VARCHAR(5) NOT NULL UNIQUE;
+ALTER TABLE survey_type_code ADD COLUMN description VARCHAR(100) NULL;
+ALTER TABLE survey_type_code ADD COLUMN active_ind BOOLEAN NOT NULL DEFAULT TRUE;
+
+
+        
+-- ### Creating Comments on table ### --
+
+        
+COMMENT ON TABLE survey_type_code IS 'Code table for schema of survey types';
```

Comment:
I'd like to recommend we add a more descriptive table comment that describes a "survey type" from a business perspective. For example; The Survey Type Code is an indication of the standard to which survey data has been collected in the field (i.e. Cursory, Operational, Precise).

Let's ask Mike.